notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>A nitpick that you can ignore, but it'd be nice to have the mentions of HST and TESS as hyperlinks to reference our STScI archive documentation of the missions: </p><p> </p><p>https://archive.stsci.edu/missions-and-data/hst</p><p>https://archive.stsci.edu/missions-and-data/tess</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>